projects/components/src/table/table.component.ts

```diff
@@ -174,9 +174,7 @@ export class TableComponent
     width: '32px',
     visible: true,
     renderer: StandardTableCellRendererType.Checkbox,
-    onClick: (row: StatefulTableRow) => {
-      this.toggleRowSelection(row);
-    }
+    onClick: (row: StatefulTableRow) => this.toggleRowSelected(row)
```

Comment:
Is this being used?

Reply:
Yes. The checkbox click actually triggers this

Reply:
ok the `ifCheckbox` escapehatch was removed, got it